README.md

```diff
@@ -33,12 +33,13 @@ actionSets:
       scope: auth-scope-a
```

Comment:
Could you add a predicate here to highlight that predicates can be used as action top level attr?